### DISCUSSION TITLE
Basic new comer questions

### DISCUSSION THREAD

**Discussion by DuvelCorp** [chrisbenincasa/tunarr#917] [Questions & Help] [answered]
Hi

I read your post on jellyfin forum and landed here. And I just discovered this thing and its ancestor(s), but I am a total noob in tuner/livetv stuff.
I've got thousands of movies and series and a dozen of (friends/family) users on my Jellyfin server. And this looks like an amazing thing to put in place.... BUT

I quickly read the concept and setup/config, and I still have "THE bottleneck" question for which I dont have an answer : 
Basically:

- Does tunarr process/transcode/stream the channels 24/7 and "in the wild", even if no one is watching ? just like an UDP endless continuous stream ???
- Or does it start to really process a stream if at least one user selected a channel in Jellyfin?

Because the former would be really way too much ressource-intensive for me.

Additional question : how does it deals with multi-language contain (Audio/Subtitles)? 80% of my stuff has FRENCH+ENGLISH. But most users wants French. Can you define the default somewhere on channel config, and/or does users still have the opportunity to change audio track and subtiles just like in normal jellyfin stuff??

Cheers

**Comment by chrisbenincasa** [comment, marked as answer]
Hey there - sorry for the delay in answering! I should checked discussions more. To answer your questions:

- Tunarr transcodes on demand (I.e. when you tune into the channel). It is not continuous 
- The 0.18.0 release includes preferences for specifying your preferred language for audio. Subtitle support is under construction but will offer the same options.
